tests/network/jumbo_frame/test_bridge.py

```diff
@@ -17,11 +17,13 @@
 )
 from utilities.virt import VirtualMachineForTests, fedora_vm_body, running_vm
 
-pytestmark = pytest.mark.usefixtures(
-    "skip_if_no_multinic_nodes",
-    "skip_when_no_jumbo_frame_support",
-    "hyperconverged_ovs_annotations_enabled_scope_session",
-)
+pytestmark = [
+    pytest.mark.special_infra,
```

Comment:
Ditto.
Unless I am wrong, and multi-NIC is considered a special infra eventually.

Reply:
Same, for jumbo not for multi-nic.

Reply:
@yossisegev, @EdDev is correct. If you need jumbo frame, it requires special config.

Reply:
Again - I don't think it's true that jumbo is considered a special configuration.
I think that most of our clusters - including PSI - support jumbo. AFAIK DevOps don't apply any special configuration in order to deploy a jumbo-supporting cluster.

Reply:
@yossisegev please check deploy jobs for BM clusters and compare it with PSI. I see JUMBO_FRAME option is only available for BM deployments.

Reply:
ditto